### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by Kallamamran** [TheCompAce/Auto-GPT-Powershell#5] [General]
Hi there,

I just stumbled on this repo by chance and wonder... Running the AutoGPT.ps1 script, what is installed?
I'm running Python, xformers and so on allready and don't want my system messed up with updated dependencies and stuff.

I will track this project for sure. Been looking for a local model to run AutoGPT :)

**Comment by TheCompAce** [comment]
This does not install anything, you down load the model for offline, I hope to expand it using local APIs and other Online ChatGPTs. But I just started the project 2 days ago.  But it is only the session log file plugin so far.

**Reply by Kallamamran** [reply]
That's interesting. I have no idea how that works since every other AI I have installed runs on python and alot of dependencies :)

**Reply by TheCompAce** [reply]
The exe used is from https://github.com/nomic-ai/gpt4all/blob/main/README.md, if you want more info.  But it simply calls the exe and send the prompt to it and it returns the data with a response in it. (Same thing for the ChatGPT online, but uses the OpenAI chat API.

**Reply by Kallamamran** [reply]
Don't quite know what you mean by "send the prompt to it and it returns the data with a response in it"? Isn't this a local model where the API is not needed? I was curious to see what the model could do... Locally

**Reply by TheCompAce** [reply]
I use the exe file as a proxy, so it works the same as a online api call from this scripts point of view.

**Reply by Kallamamran** [reply]
Well... I don't get it working anyway :) Iäm keeping at it, but no matter the start prompt it only generate red errrors

**Reply by TheCompAce** [reply]
Well you can post the errors, and try the latest code. You can also hit "y" on the change options at the start and turn on Debug.

**Reply by Kallamamran** [reply]
Posted in issues

**Comment by TheCompAce** [comment]
Updated code, and hopefully you want have to worry about the plugins not updating.
